manager.go

```diff
@@ -286,8 +287,7 @@ func (m *Manager) prunePeers() {
 			count++
 			continue
 		}
-		if !node.LastSuccess.IsZero() &&
-			now.Sub(node.LastSuccess) > pruneExpireTimeout {
+		if now.Sub(node.LastSuccess) > pruneExpireTimeout {
```

Comment:
Or perhaps this should have been with `LastAttempt` i.e. `!now.Sub(node.LastAttempt).IsZero()`?  Because without screening out these, I think an il-timed firing of `pruneAddressTicker` could prune a new peer that is presently being tested in `pruneAddressTicker`.